x/dex/types/limit_order_tranche_key.go

```diff
@@ -0,0 +1,75 @@
+package types
+
+import (
+	"errors"
+	"strings"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+var _ TickLiquidityKey = (*LimitOrderTrancheKey)(nil)
+
+func (p LimitOrderTrancheKey) KeyMarshal() []byte {
+	var key []byte
+
+	pairKeyBytes := []byte(p.TradePairID.MustPairID().CanonicalString())
+	key = append(key, pairKeyBytes...)
+	key = append(key, []byte("/")...)
+
+	makerDenomBytes := []byte(p.TradePairID.MakerDenom)
+	key = append(key, makerDenomBytes...)
+	key = append(key, []byte("/")...)
+
+	tickIndexBytes := TickIndexToBytes(p.TickIndexTakerToMaker)
+	key = append(key, tickIndexBytes...)
+	key = append(key, []byte("/")...)
+
+	liquidityTypeBytes := []byte(LiquidityTypeLimitOrder)
+	key = append(key, liquidityTypeBytes...)
+	key = append(key, []byte("/")...)
+
+	key = append(key, []byte(p.TrancheKey)...)
+	key = append(key, []byte("/")...)
+
+	return key
+}
+
+func (p LimitOrderTrancheKey) KeyUnmarshal(bz []byte) error {
```

Comment:
what is this function supposed to do?

Reply:
Unmarshal a key from a KV store bytes sequence. Currently unused but possibly used in the future.